### DISCUSSION TITLE
Timeline for Notary v2

### DISCUSSION THREAD

**Discussion by tpilli** [sse-secure-systems/connaisseur#699] [Q&A]
Is there any timeline to integrate with Notary v2?
[Website](https://sse-secure-systems.github.io/connaisseur/v2.6.0/validators/notaryv2/) says TBD, curious to know if there's any timeline.

**Comment by xopham** [comment]
@tpilli there is currently no strict timeline. When we started there was only Notary v1 and then Sigstore and Notary v2 started off as alternatives/new solutions. Since those appeared to be the major ones, we felt we should support both. However, I believe that Notary v2 is not yet widely usable, as it requires changes to the image registry which would currently mean, we'd be implementing a solution that is not generally available.

**Reply by xopham** [reply]
@tpilli are you using a registry that is already capable of Nv2? If so which? Would be excited to learn more about adoption.
